views/base.njk

```diff
@@ -9,8 +9,8 @@
     <meta name="description" value="{{doc.fields.description}}">
     <link rel="preconnect" href="https://fonts.gstatic.com">
     <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="{{pod.staticFile('/dist/css/main.min.css').url.path}}">
-    <script src="{{pod.staticFile('/dist/js/main.min.js').url.path}}?cb={{cb()}}"></script>
+    <link rel="stylesheet" href="{{a.static('/dist/css/main.css').url.path}}">
```

Comment:
Switch this back to using the "pod functions" (aka `pod.*`). We are going to remove the `a.*` ones.

Reply:
Done